src/developer/web-api/tracker.md

```diff
@@ -1492,6 +1500,48 @@ Two endpoints are dedicated to tracked entities:
 - `GET /api/tracker/trackedEntities/{id}`
   - retrieves a tracked entity given the provided id
 
+#### Response content types
+
+If not otherwise specified, JSON is the default response for the `GET` method. 
```

Comment:
Can you keep this as a paragraph. We add so many newlines in our docs that are odd.